Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -188,13 +188,14 @@ var ListView = React.createClass({
     /**
      * (sectionData, sectionID) => renderable
      *
-     * If provided, a sticky header is rendered for this section.  The sticky
-     * behavior means that it will scroll with the content at the top of the
-     * section until it reaches the top of the screen, at which point it will
-     * stick to the top until it is pushed off the screen by the next section
-     * header.
+     * If provided, a sticky header / footer is rendered for this section.
+     * The sticky behavior means that it will scroll with the content at the
+     * top or bottom of the section until it reaches the edge of the screen, at
+     * which point it will stick to the top / bottom until it is pushed off
+     * the screen by the next section header / footer.
```

Comment:
I wonder if this will render properly in the autodocs. Won't this result in renderSectionFooter having no documentation attached to it? What is displayed when you navigate to ListView.html?